src/OSPSuite.Utility/FileHelper.cs

```diff
@@ -267,15 +267,21 @@ public static string CreateRelativePath(string originalPath, string relativeToPa
          }
       }
 
-      private static string appendDirectorySeparatorChar(string path)
+      private static string appendDirectorySeparatorChar(string path, bool isRelativeToPath)
       {
-         // Append a slash only if the path is a directory and does not have a slash.
-         if (!Path.HasExtension(path) && !path.EndsWith(Path.DirectorySeparatorChar.ToString()))
-            return path + Path.DirectorySeparatorChar;
+         var endsWithSeparator = path.EndsWith(Path.DirectorySeparatorChar.ToString());
+         if (endsWithSeparator)
+            return path;
+
+         // Append a slash only if the path is a directory or the path is a file use to base the relativePath to 
+         if (!Path.HasExtension(path) || isRelativeToPath)
```

Comment:
I just realized that when resolving absolute path relative to a file (as opposed to a folder) the path needs to end with a dir separator otherwise it does not work... never used that case before I guess. But it's used a lot in the qualification